hypertrace-graphql-span-processing-schema/src/main/java/org/hypertrace/graphql/spanprocessing/schema/mutation/SpanProcessingMutationSchema.java

```diff
@@ -16,6 +17,8 @@ public interface SpanProcessingMutationSchema {
 
   @GraphQLField
   @GraphQLName(CREATE_EXCLUDE_SPAN_RULE_MUTATION_NAME)
+  @GraphQLDescription(
+      "Create exclude span rule by providing the rule name, filters, rule type, enabled or disabled state")
```

Comment:
You don't necessary have to write every field out in the rule - you've reviewed docs before I'm sure :) what would you find useful? These are for other developers to understand and craft queries. 

Here, for example you might just reference the `ExcludeSpanRule` type and put your primary documentation on that. E.g. "Creates a new ExcludeSpanRule. See ExcludeSpanRule for more information."